src/utils/Guid.ts

```diff
@@ -0,0 +1,79 @@
+import { isObject } from '@vueuse/core'
```

Comment:
What's the reason for including this much logic here? To me, there's not much value to having this and it rather makes me feel like I'm in a C# or Java project ...